### PR TITLE
ember-cli-update to 2.16

### PR DESCRIPTION
**Pull request by Dhaulagiri**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,33 @@
...
+addons:
+  chrome: stable
 
 cache:
   directories:
-    - node_modules
+    - $HOME/.npm
 
 env:
-  - EMBER_TRY_SCENARIO=ember-1.13
```
Comment (Dhaulagiri) [inline .travis.yml]: I can put back 1.13 if there is a strong desire to keep testing that scenario

**Comment by Dhaulagiri** [comment]
@rlivsey any chance we could get this in and released?

**Comment by SergeAstapov** [comment]
Hi @rlivsey!
do you think this can be merged? This PR updates `ember-cli-babel` and as a result dedup `node_modules` folder that reduces build time

**Comment by Dhaulagiri** [comment]
@SergeAstapov FWIW I know it can be hard for folks to keep on top of addons like this, so if you are still looking for an svg addon with babel 6 https://github.com/ivanvotti/ember-svg-jar has a nearly identical api as this addon which makes them pretty easy to swap

**Pull request closed without merging**